tests/JobCreatorTest.php

```diff
@@ -2138,16 +2138,16 @@ public function provideGetInstallerVersionFromComposer(): array
             // highest major
             ['myaccount/silverstripe-admin', 'mybranch', ['silverstripe/framework' => $highestMajor . '.x-dev'], 'silverstripe-module', $highestMajor . '.x-dev'],
             ['myaccount/silverstripe-admin', 'mybranch', ['silverstripe/framework' => $highestMajor . '.0.x-dev'], 'silverstripe-vendormodule', $highestMajor . '.0.x-dev'],
-            ['myaccount/silverstripe-admin', 'mybranch', ['silverstripe/framework' => '^' . $highestMajor], 'silverstripe-theme', $highestMajor . '.0.x-dev'],
-            ['myaccount/silverstripe-somemodule', 'mybranch', ['silverstripe/cms' => '^' . $highestMajor], 'silverstripe-recipe', $highestMajor . '.0.x-dev'],
```

Comment:
These were changed to dynamically use latest minor in the CMS 6 i.e. 6.1

Reply:
Seems okay